src/mds/BoostUrlImpl.cc

```diff
@@ -1,8 +0,0 @@
-/*
- * https://www.boost.org/doc/libs/1_82_0/libs/url/doc/html/url/overview.html#url.overview.requirements
- *
- * To use the library as header-only; that is, to eliminate the requirement 
- * to link a program to a static or dynamic Boost.URL library, 
- * simply place the following line in exactly one source file in your project.
- */
-#include <boost/url/src.hpp>
```

Comment:
fyi @leonid-s-usov 

the [1.85 docs](https://www.boost.org/doc/libs/1_85_0/libs/url/doc/html/url/overview.html#url.overview.quick_look.integration) now say:
> Boost.URL is a compiled library. You need to link your program with the Boost.URL built library. You must install binaries in a location that can be found by your linker.

Reply:
Great! I love it!